misago/threads/api/postingendpoint/attachments.py

```diff
@@ -31,6 +31,8 @@ class AttachmentsSerializer(serializers.Serializer):
     attachments = serializers.ListField(
         child=serializers.IntegerField(), required=False
     )
+    # Important fix. Without this I can not run
+    update_attachments = False
```

Comment:
Whats the error? You are making `update_attachments` flag class attribute, which will break `AttachmentsSerializer` behavior when multiple users are uploading at same time.

Reply:
Method save is run before validate_attachments an cause error:
AttributeError: 'AttachmentsSerializer' object has no attribute 'update_attachments'
For example in test_moderation_queue